extensions/agroal/deployment/src/test/resources/application-default-datasource.properties

```diff
@@ -8,3 +8,4 @@ quarkus.datasource.background-validation-interval=PT53S
 quarkus.datasource.acquisition-timeout=PT54S
 quarkus.datasource.leak-detection-interval=PT55S
 quarkus.datasource.idle-removal-interval=PT56S
+quarkus.datasource.transaction-isolation=SERIALIZABLE
```

Comment:
I think I would have named it `transaction-isolation-level`.

Reply:
Done.